main/Kconfig.projbuild

```diff
@@ -22,16 +22,26 @@ menu "Featured FreeRTOS IoT Integration"
         default 1 if APP_WIFI_PROV_TRANSPORT_SOFTAP
         default 2 if APP_WIFI_PROV_TRANSPORT_BLE
 
+    config GRI_RUN_QUALIFICATION_TEST
+        bool "Run qualification test."
+        default n
+        help
+            Enable to run qualification test. Use configurations in test_execution_config.h and test_param_config.h.
+            Disable to demo. Use configurations below (in sdkconfig).
+
```

Comment:
Could add the test configs here under a menu that depends on if GRI_RUN_QUALIFICATION_TEST is enabled.

Reply:
Nice catch! I'll update more configurations for qualification test later!